components/org.wso2.carbon.identity.authz.valve/src/main/java/org/wso2/carbon/identity/authz/valve/util/Utils.java

```diff
@@ -88,20 +88,15 @@ public static boolean isUserBelongsToRequestedTenant(AuthenticationContext authe
                     Constants.AUTH_CONTEXT_OAUTH_APP_PROPERTY);
             tenantDomain = OAuth2Util.getTenantDomainOfOauthApp(oAuthAppDO);
         }
-        return tenantDomainFromURLMapping.equals(tenantDomain);
-    }
-
-    public static boolean isUserAuthorizedForOrganization(AuthenticationContext authenticationContext, Request request) {
```

Comment:
When reverting this PR again if stop supporting organization qualified URLs, only revert the changes done to this method.